src/components/common/Onboarding/wizardSteps/CreateColony/CreateColonyCardRow.tsx

```diff
@@ -41,7 +41,7 @@ const CardRow = ({ updatedWizardValues, setStep }: CardRowProps) => {
     {
       title: 'navigation.admin.colonyDetails',
       text: colonyDisplayName,
-      subText: `app.colony.io/${colonyName}`,
+      subText: `app.colony.io/${colonyName.toLowerCase()}`,
```

Comment:
This should actually use the global ENV level url value for the base domain.

Can you please update that as well ?

```suggestion
      subText: `${APP_URL.origin}/${colonyName.toLowerCase()}`,
```

Reply:
🤦‍♂️ I should have caught that, updating now!